presto-hive/src/test/java/com/facebook/presto/hive/AbstractTestHiveClient.java

```diff
@@ -1962,7 +1963,7 @@ private void doTestBucketedTableEvolutionWithDifferentReadCount(HiveStorageForma
 
             ImmutableList.Builder<MaterializedRow> allRows = ImmutableList.builder();
             for (ConnectorSplit split : splits) {
-                try (ConnectorPageSource pageSource = pageSourceProvider.createPageSource(transaction.getTransactionHandle(), session, split, tableHandle.getLayout().get(), columnHandles)) {
+                try (ConnectorPageSource pageSource = pageSourceProvider.createPageSource(transaction.getTransactionHandle(), session, split, tableHandle.getLayout().get(), columnHandles, new SplitContext(false))) {
```

Comment:
Make a `public static final` constant in class `SplitContext` for `new SplitContext(false)` and use it everywhere in the codebase when necessary.